### PR TITLE
add Python examples for image_transport_py 

### PR DESCRIPTION
**Pull request by tfoldi**
* moved `image_transport_tutorials` package to `/image_transport_tutorials` subdirectory
 * fixed links in `README.md`
 * added `image_transport_tutorials_py` package with a minimal example using `image_transport_py`
 * added example walkthrough to `README.md`

### CODE REVIEW

**Comment by tfoldi** [comment]
I can update README.md in this or a separate PR

File: README.md [hunk 1]
```diff
@@ -288,3 +299,103 @@ $ ros2 param get /image_publisher jpeg_quality
...
+Take a look at [my_subscriber.py](image_transport_tutorials_py/image_transport_tutorials_py/my_subscriber.py).
+
+
+To subscribe to images, use `ImageTransport` to create a subscription to the image topic.
+
+Steps:
+
+1. Import Necessary Modules:
+
+```
```
Comment (ahcorde) [inline README.md:370]:
```suggestion
```python
```

File: image_transport_tutorials_py/test/test_copyright.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
```
Comment (ahcorde) [inline image_transport_tutorials_py/test/test_copyright.py:1]:
```suggestion
# Copyright 2024 Open Source Robotics Foundation, Inc.
```

File: image_transport_tutorials_py/test/test_flake8.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
```
Comment (ahcorde) [inline image_transport_tutorials_py/test/test_flake8.py:1]:
```suggestion
# Copyright 2024 Open Source Robotics Foundation, Inc.
```

File: image_transport_tutorials_py/test/test_pep257.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2015 Open Source Robotics Foundation, Inc.
```
Comment (ahcorde) [inline image_transport_tutorials_py/test/test_pep257.py:1]:
```suggestion
# Copyright 2024 Open Source Robotics Foundation, Inc.
```

**Review by ahcorde — APPROVED** [review]

**Pull request merged**